packages/web-client/app/components/card-pay/dashboard-panel/index.hbs

```diff
@@ -1,24 +1,41 @@
-<details
-  class={{cn "dashboard-details-container" dashboard-details-container--no-collapse=@noCollapse}}
-  ...attributes
-  open={{if @noCollapse true @open}}
->
-  <summary class="dashboard-details-container__summary">
-    <div class="dashboard-panel__summary-hero" style={{css-url "background-image" @panel.summaryHeroImageUrl}} />
-    <div class="dashboard-details-container__summary-content">
-      <h2 class="dashboard-panel__summary-title">{{@panel.title}}</h2>
-      <p class="dashboard-panel__summary-desc">{{@panel.description}}</p>
-    </div>
-    <div class="dashboard-details-container__marker">
-      {{svg-jar "caret-right" class="dashboard-details-container__marker-icon" role="presentation"}}
-    </div>
-  </summary>
-  <section class="dashboard-panel">
-    <header class="dashboard-panel__header">
-      <h2 class="dashboard-panel__header-title">{{@panel.title}}</h2>
-      <p class="dashboard-panel__header-desc">{{@panel.description}}</p>
-    </header>
-    <div class="dashboard-panel__hero" style={{css-url "background-image" @panel.heroImageUrl}} />
-    {{yield}}
-  </section>
-</details>
\ No newline at end of file
+{{#let (if @noCollapse true @open) as |isOpen|}}
+  <details
+    class={{cn
+      "dashboard-details-container"
```

Comment:
The CSS class name pattern in this file is inconsistent. Sometimes it is `dashboard-details-container` and sometimes it is `dashboard-panel`. Based on the component name, I would expect it to consistently be `dashboard-panel`

Reply:
Good suggestion - I cleaned that up.